client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -59,6 +60,12 @@ Template.incidentForm.onCreated ->
 
     @incidentStatus.set(@incidentData.status or '')
 
+    @locations.set(@incidentData.locations.map (loc)->
+      id: loc.id
+      text: formatLocation(loc)
+      item: loc
+    )
```

Comment:
Just suggesting some minor code style updates: remove the surrounding parens and add a space after the arguments?